servers/republik/modules/crowdfundings/lib/CustomPackages/rules/endTooFarAway.js

```diff
@@ -0,0 +1,33 @@
+const debug = require('debug')('crowdfundings:lib:CustomPackages:rule:endTooFarAway')
+
+const { getLastEndDate } = require('../../utils')
+
+const HORIZON_MONTHS = 3
```

Comment:
Let's ensure all march people have to prolong too.
```suggestion
const HORIZON_MONTHS = 4
```